onnxruntime/test/testdata/mobilenet_v3_small_excerpt_gen.py

```diff
@@ -0,0 +1,119 @@
+"""
+Run this script to recreate the original onnx model.
+Example usage:
+python mobilenet_v3_small_excerpt_gen.py out_model_path.onnx
```

Comment:
nit: might be useful to mention you ran onnx2py on the full model and stripped things down to the desired nodes in case someone wants to generate an excerpt from a different model.

Reply:
and now I've learnt something. wasn't aware of onnx.untils.extract model.